packages/react/src/components/dateInput/DateInput.stories.tsx

```diff
@@ -209,3 +211,61 @@ Success.args = {
   id: 'Success',
   successText: 'Date is valid',
 };
+
+export const WithCustomDayStyles = (args: DateInputProps) => {
+  const dateFormat = 'dd.M.yyyy';
+  const dateValue = new Date(2021, 10, 12);
+  const [value, setValue] = useState<string>(format(dateValue, dateFormat));
+  const helperText = `Custom styles for days with limited available timeslots in the date picker calendar. Use format D.M.YYYY.`;
+
+  const littleSpaceLeftDate: LegendItem = {
+    elementId: 'little-space-left',
+    label: 'Only a few free timeslots available',
+    color: 'var(--color-summer-medium-light)',
+  };
+
+  const dayHasLittleSpace = (day: number) =>
+    day === 3 || day === 4 || day === 17 || day === 19 || day === 23 || day === 24;
+
+  const getDateLegendId = (date: Date) => {
+    const day = date.getDate();
+    if (dayHasLittleSpace(day)) return littleSpaceLeftDate.elementId;
+    return undefined;
+  };
+
+  const setDateClassName = (date: Date) => {
+    const legendId = getDateLegendId(date);
+    if (legendId === littleSpaceLeftDate.elementId) return 'little-space-left';
+    return undefined;
+  };
+
+  const setDateAriaDescribedBy = (date: Date) => {
+    return getDateLegendId(date);
+  };
+
+  const legend = [littleSpaceLeftDate];
+  return (
+    <>
+      <style>
+        {`
+          .little-space-left {
+              --date-background: var(--color-summer-medium-light);
+              --date-border: 1px solid black;
+              --horizontal-spacing: 1px;
```

Comment:
`horizontal-spacing` is only used in `hds-datepicker` element's padding. Setting it in here has no effect, because this is a child of that element and out of scope. Tried to remove this and also set it to 100px, but either has no effect, so this can be removed?

Reply:
Thanks, fixed now.